packages/liveblocks-react/src/types.ts

```diff
@@ -33,6 +33,7 @@ export type UseThreadsOptions<TThreadMetadata extends BaseMetadata> = {
   query?: {
     metadata?: Partial<TThreadMetadata>;
   };
+  scrollOnLoad?: boolean; // (optional) boolean flag to enable or disable automatic scrolling to a comment based on the URL hash when threads are loaded. Defaults to true.
```

Comment:
We should use JSDoc (and add a comment to `query` as well) so that it shows this description on hover.